src/test/java/it/pagopa/swclient/mil/controller/model/PagoPaConfTest.java

```diff
@@ -1,21 +1,79 @@
 package it.pagopa.swclient.mil.controller.model;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import jakarta.validation.Validation;
-import jakarta.validation.Validator;
+import it.pagopa.swclient.mil.util.TestBase;
 import org.junit.jupiter.api.Test;
 
-class PagoPaConfTest {
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-  Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+class PagoPaConfTest extends TestBase {
```

Comment:
same as before, don't abuse inheritance

Reply:
Resolved in [https://github.com/pagopa/mil-terminal-registry/pull/25/commits/8c55df2932810878f744b7992b7a535c9c819587](https://github.com/pagopa/mil-terminal-registry/pull/url)